frame/elections-phragmen/src/lib.rs

```diff
@@ -113,6 +114,42 @@ type BalanceOf<T> =
 type NegativeImbalanceOf<T> =
 	<<T as Trait>::Currency as Currency<<T as frame_system::Trait>::AccountId>>::NegativeImbalance;
 
+
+/// Helper functions for migrations of this module.
+pub mod migrations {
+	use super::*;
+	use frame_support::migration::StorageKeyIterator;
+	use frame_support::Twox64Concat;
+
+	/// Migrate from the old legacy voting bond (fixed) to the new one (per-vote dynamic).
+	///
+	/// Will only be triggered if storage version is V1.
+	pub fn migrate_to_recorded_deposit<T: Trait>(old_deposit: BalanceOf<T>) -> Weight {
+		if <Module<T>>::pallet_storage_version() == StorageVersion::V1 {
+
+			<StorageKeyIterator<T::AccountId, (BalanceOf<T>, Vec<T::AccountId>), Twox64Concat>>::new(
+				b"PhragmenElection",
```

Comment:
I'd really like to not rely on Strings here and get the module name from `decl_storage`. There's probably a good way for it.